ports/glutin/window.rs

```diff
@@ -233,7 +241,13 @@ impl Window {
 
     fn handle_window_event(&self, event: glutin::Event) -> bool {
         match event {
-            Event::KeyboardInput(element_state, _scan_code, Some(virtual_key_code)) => {
+            Event::ReceivedCharacter(ch) => {
+                assert!(self.pending_key_event_char.get().is_none());
```

Comment:
This assertion panics if I try to type a character using the Compose key (e.g. Compose+A+E to type `æ`).